### PR TITLE
Expose Context

### PR DESCRIPTION
**Pull request by EloB**
They added a pretty neat context thing in React. By just setting a `static contextType` to a `createContext` instance you will receive the value in `this.context`. Then you don't need to wrap your component with a Consumer. :)

https://reactjs.org/docs/context.html#when-to-use-context.

```js
const SomeContext = createContext("Joker");

class SomeComponent extends React.Component {
  static contextType = SomeContext;

  componentDidMount() {
    console.log(this.context) // Will output "Batman"
  }
}

render(
  <SomeContext.Provider value="Batman">
    <SomeComponent />
  </SomeContext.Provider>,
  something
);
```

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,15 +2,15 @@ import { TYPES, PixiComponent } from './utils/element'
...
+import { AppProvider, AppConsumer, withPixiApp, Context } from './stage/provider'
 
 /**
  * -------------------------------------------
  * Public API
  * -------------------------------------------
  */
 
-export { render, Stage, AppProvider, AppConsumer, withPixiApp, PixiComponent, PixiFiber }
+export { render, Stage, AppProvider, AppConsumer, withPixiApp, PixiComponent, PixiFiber, Context }
```
Comment (inlet) [inline src/index.js]: Let's rename it to `AppContext`

**Comment by EloB** [comment]
I will fix when I have time! 😄

File: src/index.js (hunk 1 above)
Comment (EloB) [reply src/index.js]: Fixed!

Comment (inlet) [reply src/index.js]: Awesome! Can we add a simple test for this?

Comment (EloB) [reply src/index.js]: I will add it when I have time :)

Comment (inlet) [reply src/index.js]: Awesome! I'll merge it once it's ready

**Comment by inlet** [comment]
Hi, I’d like to merge this PR 💫

Looking forward to your test to consolidate this implementation?

Cheers

**Comment by inlet** [comment]
I'm implementing React hooks now, and I need to be able to use the Context as well. Is it ok that I merge this PR and write a simple test myself? or are you perhaps writing a test at this very moment?

**Comment by EloB** [comment]
Oh nice! Hooks looks really promising. I haven't had time to implement any tests at the moment. So feel free to do that in the hooks branch and maybe close this one?

**Pull request merged**